Apps/Sandcastle/gallery/Camera.html

```diff
@@ -108,14 +108,15 @@
 
     // View in east-north-up frame
     var camera = viewer.camera;
-    Cesium.Matrix4.clone(transform, camera.transform);
     camera.constrainedAxis = Cesium.Cartesian3.UNIT_Z;
 
     // Zoom in
-    camera.lookAt(
-        new Cesium.Cartesian3(-120000.0, -120000.0, 120000.0),
-        Cesium.Cartesian3.ZERO,
-        Cesium.Cartesian3.UNIT_Z);
+    camera.lookAt({
+        target : center,
+        offset : new Cesium.Cartesian3(-120000.0, -120000.0, 120000.0),
+        transform : transform
+    });
+    camera.setTransform(transform);
```

Comment:
Err, you'll have to explain what each of these do.  The reference doc is too in the mud.  I suspect that we need some redesign across the `camera` as some things are no longer needed and there is probably more state than necessary.